test/integration/concierge_jwtauthenticator_status_test.go

```diff
@@ -0,0 +1,391 @@
+// Copyright 2024 the Pinniped contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"go.pinniped.dev/generated/latest/apis/concierge/authentication/v1alpha1"
+	"go.pinniped.dev/test/testlib"
+)
+
+// Never run this test in parallel since deleting all federation domains is disruptive, see main_test.go.
+func TestConciergeJWTAuthenticatorStatus_Disruptive(t *testing.T) {
```

Comment:
This comment is wrong. Also, could this test be `_Parallel` instead of `_Disruptive`?

Reply:
Fishing up the logic now that dictates what `_Parallel` and `_Disruptive` mean.  I'm sure you are correct, I'll just go remind myself & then push a change.